### PR TITLE
a few more suggestions

### PR DESCRIPTION
**Pull request by adamjonas**

### CODE REVIEW

**Comment by billygarrison** [comment]
I had already written my changes and was about to submit my PR when I saw you also made some changes here. So I included your changes in my PR (https://github.com/renepickhardt/The-Lightning-Network-Book/pull/7) since mine had a lot more changes. Otherwise I would have edited this PR.

Sorry if this is bad practice but I am new to git/GitHub.

File: lightning-network-book.tex [hunk 1]
```diff
@@ -111,12 +111,12 @@ \section{Introduction to Electronic Cash}
...
 
 We can conclude that electronic cash is only useful if we solve the problem of double spending electronic coins and create a limited supply of electronic coins.
 
 One obvious solution is a central authority or custodian who understands who owns which coins and who allows cash transactions only if the sender has enough cash.
-While this solution works and is already implemented by traditional banks it does not solve our desire that we carry around our electronic coins in our devices and - even worse - we still need to trust third-party service with all the risks that we have experienced with the collapse of Lehman Brothers.
+While this solution works and is already implemented by traditional banks, it does not solve our desire that we carry around our electronic coins in our devices and - even worse - we still need to trust third-party service with all the risks that we have experienced with the collapse of Lehman Brothers.
 
 Luckily in late 2008 a person or group under the pseudonym Satoshi Nakamoto published the Bitcoin paper.
```
Comment (adamjonas) [inline lightning-network-book.tex]: add link to white paper?

Comment (renepickhardt) [reply lightning-network-book.tex]: will happen at some point in time but it is not clear that the paper is mentioned here for the first time

File: lightning-network-book.tex [hunk 2]
```diff
@@ -40,7 +40,7 @@ \chapter{Introduction}
 We continue by explaining the theory and construction of the Lightning Network.
 We will understand how to construct payment channels via Revocable Sequence Maturity Contracts and how to solve the routing issues via Hashed Time Locked Contracts.
 These two forms of smart contracts allow for the Blockchain to be transformed from a transaction layer for payments to an enforcing layer of those contracts.
-Similar to the real world most contracts have never to be enforced via a court ruling but are negotiated bilaterally between consenting parties.
+Similar to the real world, most contracts will never to be enforced via a court ruling but are negotiated bilaterally between consenting parties.
```
Comment (renepickhardt) [inline lightning-network-book.tex]: is that actually true? When including the word `will` I would write: 
... will never  be enforced...
omitting the word to

**Comment by renepickhardt** [comment]
will close as this is resolved by RP #7 anyway

File: lightning-network-book.tex [hunk 3]
```diff
@@ -40,7 +40,7 @@ \chapter{Introduction}
 We continue by explaining the theory and construction of the Lightning Network.
 We will understand how to construct payment channels via Revocable Sequence Maturity Contracts and how to solve the routing issues via Hashed Time Locked Contracts.
 These two forms of smart contracts allow for the Blockchain to be transformed from a transaction layer for payments to an enforcing layer of those contracts.
-Similar to the real world, most contracts will never to be enforced via a court ruling but are negotiated bilaterally between consenting parties.
+Similar to the real world, most contracts will never have to be enforced via a court ruling but are negotiated bilaterally between consenting parties.
```
Comment (renepickhardt) [inline lightning-network-book.tex]: haha sorry too late and @billygarrison has corrected this while merging anyway. I merged the other one so you already have your attribution. hope that is a fine process for you and thanks for your support!

**Pull request closed without merging**